tools/pylint-build-scan.py

```diff
@@ -37,9 +37,10 @@
 PYLINTRC = os.path.join(SCRIPTPATH, 'pylintrc')
 FILENAMES = [
     os.path.abspath(__file__),
-    '{sitepackages}/pki',
+    os.path.abspath("base/server/healthcheck/setup.py")
 ]
 UPGRADE_SCRIPT = re.compile('^[0-9]+-.*')
+PYTHON_SCRIPT = re.compile('\\S+\\.py$')
```

Comment:
Just FYI, the `UPGRADE_SCRIPT` probably can be replaced with `PYTHON_SCRIPT` since all upgrade scripts now end with `.py`, but we can leave that for future clean up if you want.

Reply:
ACK. I have updated this PR.